apps/dapp-console/app/constants/envVars.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod'
+
+const schema = z.object({
+  GROWTHBOOK_ENDPOINT: z
+    .string()
+    .url()
+    .default('https://cdn.growthbook.io/api/features/')
+    .describe('Growthbook endpoint'),
+  GROWTHBOOK_ENCRYPTION_KEY: z
+    .string()
+    .optional()
+    .describe('Key for decrypting growthbook response'),
+})
+
+export const ENV_VARS = schema.parse({
```

Comment:
we won't be able to pass `process.env` into a function like we do in gateway to generate the values at runtime, we need to do it statically at buildtime with next